src/cmd/prepare.go

```diff
@@ -72,7 +72,8 @@ var prepareComputeFileSha256sum = &cobra.Command{
 var prepareFindImages = &cobra.Command{
 	Use:     "find-images",
 	Aliases: []string{"prep"},
-	Short:   "evaluates components in a zarf file to identify images specified in their helm charts and manifests",
+	Short: "Evaluates components in a zarf file to identify images specified in their helm charts and manifests.\n" +
+		"Requires the `--repo-chart-path` flag to be provided.",
```

Comment:
this flag is optional, but needed for big bang and should be "-p ./chart".  Technically could be for any helm chart hosted in git, but we focus on Big Bang

Reply:
Thanks, that makes sense. I updated the message (as well as corrected the message type).